travis/start_scheduler.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -ev
 
+cd $PROJECT_DIR/../executor
+bin/build-cook-executor.sh
+cp -fv $PROJECT_DIR/../executor/dist/cook-executor $PROJECT_DIR/../travis/
+mkdir -p $PROJECT_DIR/../scheduler/resources/public/
+cp -fv $PROJECT_DIR/../executor/dist/cook-executor $PROJECT_DIR/../scheduler/resources/public/
```

Comment:
Why does this need file need to go to both `$PROJECT_DIR/../travis/` and `$PROJECT_DIR/../scheduler/resources/public/`?